test/components/selection-list.spec.tsx

```diff
@@ -10,6 +10,10 @@ import {
 import {SelectionListTestDriver} from '../../test-kit';
 import {sleep, WithTheme, WithThemeDAID} from '../utils';
 
+function focusedIndex(list: SelectionListTestDriver) {
```

Comment:
why not have this method in the `selectionListDriver` itself? as in `list.getFocusedItemIndex()`